content/docs/command-reference/remote/modify.md

```diff
@@ -328,19 +328,26 @@ storage. Whether they're effective depends on each storage platform.
 
 By default, DVC authenticates using an Azure
 [default credential](https://docs.microsoft.com/en-us/python/api/azure-identity/azure.identity.defaultazurecredential)
-(if any). This uses certain environment variables or other auth sources. Use the
-following parameters (listed in order or precedence) to customize the
-authentication method:
+(if any). This uses certain environment variables or other auth sources. The
+following parameters (listed in order of precedence) are used for
+authentication:
```

Comment:
I updated this wording btw, hope it's ok.

Reply:
Actually @casperdcl, the rewording here changed the semantics. These params are only used **if set** (a default credential is the used otherwise, as indicated at the beginning of the paragraph). Not 100% sure though, please see https://github.com/iterative/dvc.org/pull/2489#discussion_r643676441.

Reply:
Fixed in 843ff79.